tree/ntuple/v7/inc/ROOT/libdaos_mock/daos.h

```diff
@@ -254,37 +280,35 @@ typedef struct {
 
 /** Container information */
 typedef struct {
-	char		unused; // silence [-Wextern-c-compat]
+   uuid_t ci_uuid;
+   char unused; // silence [-Wextern-c-compat]
```

Comment:
This member was only added to silence the `-Wextern-c-compat`, where 0-sized objects are not required to have different addresses.  Thus, we can remove it now. 
```suggestion
```